### PR TITLE
Register evaluation data writer

### PR DESCRIPTION
**Pull request by giuliano-97**
Register the evaluation data writer with config utils factoryRos 
so that it can be configured from a YAML file.

### CODE REVIEW

File: panoptic_mapping/src/tools/evaluation_data_writer.cpp [hunk 1]
```diff
@@ -11,19 +11,22 @@
...
 
-void EvaluationDataWriter::Config::checkParams() const {}
-
 EvaluationDataWriter::EvaluationDataWriter(const Config& config)
-    : LogDataWriter(config_.log_data_writer_config, false),
-      config_(config.checkValid()) {
+    : config_(config.checkValid()),
+      LogDataWriter(config.log_data_writer_config, false) {
   LOG_IF(INFO, config_.verbosity >= 1) << "\n" << config_.toString();
-  // NOTE(schmluk): Setup is invoked from the LogDataWriter
```
Comment (giuliano-97) [inline panoptic_mapping/src/tools/evaluation_data_writer.cpp:26]: The ctor of LogDataWriter just calls the setup method of the parent class. The one of the derived class has to be called to enable saving the maps.

**Comment by giuliano-97** [comment]
@Schmluk not sure if this is already fixed in another branch - if so, feel free to close the PR :)

File: panoptic_mapping/src/tools/evaluation_data_writer.cpp (hunk 1 above)
Comment (Schmluk) [reply panoptic_mapping/src/tools/evaluation_data_writer.cpp:26]: True, good catch! This was changed in a bit of a rush and not properly tested apparently (both data writers used to be the same class before).

**Review by Schmluk — APPROVED** [review]

**Comment by Schmluk** [comment]
LGTM, good that you saw these and good fixes!

**Pull request merged**